tests/testthat/test-get.R

```diff
@@ -360,3 +360,39 @@ test_that("get_packages() works as expected with `content_guid` names in API res
     )
   )
 })
+
+test_that("get_content only requests vanity URLs for Connect 2024.06.0 and up", {
+  with_mock_dir("2024.05.0", {
+    client <- Connect$new(server = "http://connect.example", api_key = "not-a-key")
+    client$version
```

Comment:
`version` is fetched lazily, so calling it here just ensure that it's fetched from the correct mock dir before we `expect_GET` within `without_internet()`.

Reply:
How about this, to memorialize that comment and also be explicit:

```suggestion
    # Call $version here because it is fetched lazily, and we need it populated
    # before we do `without_internet()`
    expect_equal(client$version, "2024.05.0")
```

If you agree, you'll want to do the same for the other two tests

Reply:
Yeah — if it was worth a comment for the reviewer, probably worth a comment for future editors of the code. :)